<?xml version="1.0" encoding="UTF-8"?><Posts><Post><PostId>22133687</PostId><Url>http://stackoverflow.com/questions/22133687</Url><Stack><Type>CommunicationError</Type><Message>cannot connect: [Errno 10061] No connection could be made because the target machine actively refused it</Message><Frame><File>C:\LASTNO\Python Projects\PiBot\RobotServer\PyroServer\pyroServer.py</File><Line>63</Line><Method>doCallback</Method></Frame><Frame><File>C:\Python27\lib\site-packages\Pyro4\core.py</File><Line>160</Line><Method>__call__</Method></Frame><Frame><File>C:\Python27\lib\site-packages\Pyro4\core.py</File><Line>286</Line><Method>_pyroInvoke</Method></Frame><Frame><File>C:\Python27\lib\site-packages\Pyro4\core.py</File><Line>371</Line><Method>__pyroCreateConnection</Method></Frame></Stack></Post></Posts>